version/version.go

```diff
@@ -12,6 +12,6 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-package main
+package version
 
-const Version = "0.5.3+git"
+var Version = "0.5.3+was-not-built-properly"
```

Comment:
The motivation for this was to make the go get case a bit cleaner.
On Oct 21, 2015 18:24, "Eugene Yakubovich" <notifications@github.com> wrote:

> In version/version.go
> <https://github.com/coreos/flannel/pull/349#discussion_r42703163>:
>
> >
> > -const Version = "0.5.3+git"
> > +var Version = "0.5.3+was-not-built-properly"
>
> So this can go back to 0.0.0+was-not-built-properly
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/coreos/flannel/pull/349/files#r42703163>.
>